core/log/metric/reader.go

```diff
@@ -174,7 +175,7 @@ func readLine(bufReader *bufio.Reader) (string, error) {
 }
 
 func getLatestSecond(items []*base.MetricItem) uint64 {
-	if items == nil {
+	if items == nil || len(items) == 0 {
```

Comment:
len(items) == 0  is OK.
The length of nil slice is 0.

Reply:
Yes, the length of nil slice is 0. 
However, the zero value for slice is nil. It seems necessary to standardize the way to return an empty slice in the project. Even in the same function, there are different ways to return empty slice, such as function ReadMetricsByEndTime in this file.

Reply:
Yes, your idea is great.
Could you please d push a PR to enhance.